dolfyn/time.py

```diff
@@ -8,14 +9,32 @@ def _fullyear(year):
     return year
 
 
-def epoch2date(ds_time, offset_hr=0, to_str=False):
+def epoch2dt64(ep_time, ):
+    out = np.array(ep_time).astype('datetime64[s]')  # assumes t0=1970-01-01 00:00:00
+    out = out + ((ep_time % 1) * 1e9).astype('timedelta64[ns]')
+    return out
+
+
+def dt642epoch(dt64):
+    return dt64.astype('datetime64[ns]').astype('int') / 1e9
```

Comment:
Was having issues on particular datafiles for some reason that are fixed with:
`epoch = dt64.astype('datetime64[ns]').astype('float') / 1e9`
`return epoch.astype('int')`
Not entirely sure why.

Reply:
Converting to 'int' chops off the nanosecond data. Looking for a better approach.

Reply:
Solved in my branch of "timestamps" (https://github.com/jmcvey3/dolfyn/commits/timestamps)

Reply:
@jmcvey3 is this resolved with one of your commits, or is this still outstanding?